.vib/cloudnative-pg/goss/goss.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright Broadcom, Inc. All Rights Reserved.
+# SPDX-License-Identifier: APACHE-2.0
+
+http:
+  http://cnpg-webhook-service:{{ .Vars.service.ports.webhook }}:
+    status: 400
```

Comment:
Why forcing a "bad request" error?

Reply:
Because it's a webhook to be used by kube-apiserver, not by a regular user.

Reply:
Changed